### DISCUSSION TITLE
Smartwatch doesn't make a sound for xDrip BG alarms even though it should - possible issue with the way xDrip relays notifications to the OS?

### DISCUSSION THREAD

**Discussion by DrS0mmer** [Navid200/xDrip#102] [Alerts]
Hi all,

I've got my Galaxy Watch Active 2 set up so that it vibrates and makes a beeping sound whenever I receive a notification. And it works as intended - most of the time.  
For some reason, the watch doesn't make a sound when it displays a BG warning notification from xDrip. It still vibrates, but it doesn't beep.   
Other notifications coming from xDrip, e.g. the "persistent high" warning, work just fine.

I'm running xdrip 17cbcf0-2021.03.08 on a Google Pixel 6 with Android 12.
On my previous phone, a Huawei P20 Pro with Android 10, things worked just fine.
I'd blame this on the Galaxy Wear app, if it hadn't worked so flawlessly on my old phone. 
I'd blame this on Android 12, assuming that Google changed something in the way the OS handles notifications - if it weren't for the fact that _some_ xDrip notifications work just fine. This makes me wonder if the issue lies somewhere within xDrip?

(Apologies if I've missed any crucial information, I'm completely new to this whole Github thing!)

**Comment by Navid200** [comment]
@JasonJShuler Any ideas?
Thanks in advance

**Comment by Navid200** [comment]
We have discussions now opened in the main repository:
https://github.com/NightscoutFoundation/xDrip/discussions

I am looking for a way to transfer the open discussions from here to there. But, I'm not sure I'll succeed. In that case, it may be easier to open new discussions there.
